SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# GitHub Readme Stats Security Policies and Procedures <!-- omit in toc -->
+
+This document outlines security procedures and general policies for the
+GitHub Readme Stats project.
+
+- [Reporting a Vulnerability](#reporting-a-vulnerability)
+- [Disclosure Policy](#disclosure-policy)
+
+## Reporting a Vulnerability 
+
+The GitHub Readme Stats team and community take all security vulnerabilities
+seriously. Thank you for improving the security of our open source 
+software. We appreciate your efforts and responsible disclosure and will
+make every effort to acknowledge your contributions.
+
+Report security vulnerabilities by emailing the GitHub Readme Stats team at:
+
+```
+hazru.anurag@gmail.com
```

Comment:
Although having a security policy is a nice addition, and I understand why in high stake projects, security issues are communicated in private over email, I think for this project, it is enough to report security issues under https://github.com/anuraghazra/github-readme-stats/issues. However, if @anuraghazra is okay with having his email here, I'm OK with the merge.

Reply:
I suggest this email because it was already published inside `CODE_OF_CONDUCT.md`, see: https://github.com/anuraghazra/github-readme-stats/blob/master/CODE_OF_CONDUCT.md#enforcement

But maybe it's really better to wait @anuraghazra's approve.

Reply:
@rickstaa Alternatively we can create separate email box with shared access between all GRS team members. May be it will be better because Anurag or any other member can be busy at work and unable to take attention on vulnerability report operatively. What do you think?

Reply:
@qwerty541 Given that @anuraghazra published the email himself (I checked the commit log), I think merging this is fine. @anuraghazra can always change or remove the email later 👍🏻.